src/test/librbd/test_Groups.cc

```diff
@@ -70,6 +70,42 @@ TEST_F(TestGroup, group_createPP)
   ASSERT_EQ(0U, groups.size());
 }
 
+TEST_F(TestGroup, group_get_id)
+{
+  rados_ioctx_t ioctx;
+  rados_ioctx_create(_cluster, _pool_name.c_str(), &ioctx);
+  BOOST_SCOPE_EXIT(ioctx) {
+    rados_ioctx_destroy(ioctx);
+  } BOOST_SCOPE_EXIT_END;
+
+  librbd::RBD rbd;
+  ASSERT_EQ(0, rbd_group_create(ioctx, "group_get_id"));
+  
+  size_t size = 0;
+  ASSERT_EQ(-ERANGE, rbd_group_get_id(ioctx, "group_get_id", NULL, &size));
+  ASSERT_GT(size, 0);
+
+  char group_id[32];
+  ASSERT_EQ(0, rbd_group_get_id(ioctx, "group_get_id", group_id, &size));
+  ASSERT_EQ(strlen(group_id) + 1, size);
+  ASSERT_EQ(0, rbd_group_remove(ioctx, "group_get_id"));
```

Comment:
Add a blank line here to separate the test logic and cleanup, similar to the C++ test:
```suggestion
  
  ASSERT_EQ(0, rbd_group_remove(ioctx, "group_get_id"));
```